starknet_devnet/blocks.py

```diff
@@ -177,7 +172,7 @@ async def get_state_update(
 
             return await self.origin.get_state_update(block_hash=numeric_hash)
 
-        # now we get state updates by block_hash or from the origin
+        # now an int
```

Comment:
Isn't it latest here? Since the check before this checks if it's NOT latest

Reply:
LOL, got it, now I understand what you meant. Fixed in a40d6913d8d1842cb29473edfd3982b39ea76c99.